pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -70,7 +72,9 @@ stdenv.mkDerivation {
   # so that `SPARSE_FLAGS' corresponds to the current architecture...
   #doCheck = true;
 
-  installFlags = "NO_INSTALL_HARDLINKS=1";
+  installFlags = "NO_INSTALL_HARDLINKS=1"
+    + (if withpcre2 then " USE_LIBPCRE2=1" else "");
```

Comment:
No, libpcre (1) is in the closure ATM.  Version 2 is rather new and rarely used so far.